src/classlibnative/bcltype/stringnative.h

```diff
@@ -85,10 +85,8 @@ class COMString {
     static FCDECL2(VOID,        FCSetTrailByte,    StringObject* thisRefUNSAFE, UINT8 bData);
 #endif // FEATURE_COMINTEROP
 
-#ifdef FEATURE_RANDOMIZED_STRING_HASHING
     static FCDECL3(INT32, Marvin32HashString, StringObject* thisRefUNSAFE, INT32 strLen, INT64 additionalEntropy);
     static BOOL QCALLTYPE UseRandomizedHashing();
```

Comment:
Cleaning it up should fix the crossgen build break. If not, you can use `CROSSGEN_COMPILE` ifdef to ifdef out whatever is causing crossgen build to fail.